docs/spfx/sharepoint-2016-support.md

```diff
@@ -1,69 +1,69 @@
-# SharePoint Framework development with SharePoint 2016 Feature Pack 2
+# SharePoint Framework Development with SharePoint 2016 Feature Pack 2
```

Comment:
Global: FYI, our style is to use sentence casing for titles and headings, so "Development" here should be lowercase. This applies to capping of other headings in the file. Thanks!